src/main/java/io/openshift/launchpad/ui/booster/ProjectInfoStep.java

```diff
@@ -115,7 +115,10 @@ public void initializeUI(UIBuilder builder) throws Exception
             Set<Version> versions = catalogService.getVersions(mission, runtime);
             if (versions != null && !versions.isEmpty()) {
                runtimeVersion.setValueChoices(versions);
-               runtimeVersion.setItemLabelConverter(Version::getName);
+               runtimeVersion.setItemLabelConverter((v) -> {
+                   Booster booster = catalogService.getBooster(mission, runtime, v).get();
+                   return booster.runtimeVersionName(v.getKey());
```

Comment:
After this line: https://github.com/openshiftio/booster-catalog-service/blob/master/src/main/java/io/openshift/booster/catalog/BoosterCatalogService.java#L255